sphinx/themes/basic/static/searchtools.js

```diff
@@ -333,7 +333,7 @@ const Search = {
     _removeChildren(document.getElementById("search-progress"));
 
     const queryLower = query.toLowerCase().trim();
-    for (const [title, foundTitles] of Object.entries(allTitles)) {
+    for (const [title, foundTitles] of allTitles) {
```

Comment:
This (the whole file) actually looks a lot better to my eye.

Reply:
Yep, I'd agree with that.  Ideally I'd prefer if we could use null-prototype objects so that the presence-tests can become `x in obj` instead of `obj.hasOwnProperty(x)` -- but perhaps that'll be possible again in future.